### PR TITLE
small docs changes

### PR DESCRIPTION
**Pull request by der-john**
This PR combines two topics:

1) Docs on contexts weren't up-to-date.
2) I personally found the docs on the translation too little. I also added transifex to gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -77,3 +77,4 @@ RELEASE-VERSION
 *-requirements.txt
 /src/Mako/
 /styleguides
+.tx
```
Comment (xi) [inline .gitignore]: The transifex config is tracked by git. Why do you want it to be in gitignore?

File: docs/frontend/translation.rst [hunk 2]
```diff
@@ -52,15 +60,24 @@ can use the following command::
...
+We generally use `transifex`_ for translation. Note that new strings
+in the code are not automatically displayed in transifex, and
+updates in transifex are not instantly reflected in either the code
+or any running platform. 
+
+In order to push changes from the foo project code to transifex,
+the transifex-client can be used like this::
+
+   $ cd src/foo/
+   $ tx push -a
```
Comment (xi) [inline docs/frontend/translation.rst]: If you are not careful, you might end up overwriting new translations on transifex. My personal workflow is to pull first, manually review the changes and push afterwards. We should at lease add a warning.

File: docs/frontend/translation.rst [hunk 3]
```diff
@@ -85,19 +102,19 @@ the transifex-client can be used::
...
 informal (Du) one.
 
-All translations should use the informal variant.  The script
-``bin/change_german_salutation`` can be used to convert informal
-translations to formal ones.  Note that you will need to extend that
-script whenever the translation changes. The common workflow for this
-is: Iteratively run the script, check the output and add new rules until
-everything is fine.
+All translations should use the informal variant, except on the site
+mein.berlin.de. We use the script ``bin/change_german_salutation`` to
```
Comment (xi) [inline docs/frontend/translation.rst]: All german translations on transifex should still be informal, even for mein.berlin.de.

Also: I would like to keep specific projects out of the general adhocracy documentation.

Comment (der-john) [inline docs/frontend/translation.rst]: But, for me, without this addition, it is unclear what the script exists for.

**Pull request merged**